packages/ui-signer/src/Modal.tsx

```diff
@@ -189,6 +201,21 @@ class Signer extends React.PureComponent<Props, State> {
     });
   }
 
+  onKeyDown = (event: React.KeyboardEvent<Element>): void => {
+    if (event.keyCode === 13) {
+      const node = this.submitButtonDiv.current;
+
+      if (node instanceof HTMLElement) {
+        const child = node.firstChild;
+
+        if (child instanceof HTMLElement) {
+          child.click();
```

Comment:
Would the simpler solution rather to just do "onSend" here? Instead of 

1. making a reference
2. clicking that?

Reply:
EDIT: I just typed this up elsewhere to explain a bit better.

Instead of -

1. get ref to component and storing it
2. checking for char 13
3. creating an event on that component
4. the event calling the associated handler

We can just do -

1. check for char 13
2. directly call handler

Reply:
Absolutely!